### ISSUE TITLE
uninitialized constant RSpec::Matchers

### ISSUE THREAD

**Issue by caseyprovost** [jsonapi-rb/jsonapi-rspec#11]
On rails 5.2.3 I am getting `uninitialized constant RSpec::Matchers`

**Comment by lucasm-iRonin** [comment]
@caseyprovost I've encountered the same error. I've solved by adding `require: false` to Gemfile:

```ruby
group :test do
  ...
  gem "jsonapi-rspec", require: false
  ...
end
```

And requiring `jsonapi-rspec` manually in `spec/rails_helper.rb` file:

```ruby
require "rspec/rails"
require "jsonapi/rspec"
```

**Comment by joel** [comment]
This is already fixed on the pending PRs @beauby is there a way to merge those PRs and release a new version of this broken gem? Do this gem still active? TIA

**Comment by stas** [comment]
Closed as of #8
